.env

```diff
@@ -0,0 +1 @@
+PYTHONPATH=.
```

Comment:
This should not be included in this PR I think.

Reply:
That was an accident. I have a separate PR to add this file to .gitignore.

Reply:
Thanks for the help with this PR. Hopefully, it will make it into 0.111.